nni/retiarii/operation_def/torch_op_def.py

```diff
@@ -1,5 +1,13 @@
 from ..operation import PyTorchOperation
 
+class relu(PyTorchOperation):
+    def to_init_code(self, field):
+        return ''
+
+    def to_forward_code(self, field, output, *inputs) -> str:
+        assert len(inputs) == 1
+        return f'{output} = nn.functional.relu({inputs[0]})'
```

Comment:
in the current implementation, operation's forward code is put in operation.py

Reply:
For general `PyTorchOperation` ops, we can put them in `operation.py`. We can also allow some special ops to override `PyTorchOperation`. In the current implementation, I introduce a `ToDevice` op for changing the device placement of a module, which is required in cross-graph optimization.